src/test/clojure/clara/test_rules.clj

```diff
@@ -2361,7 +2361,22 @@
                   (query check-exception))
               (is false "Scope binding test is expected to throw an exception.")
               (catch Exception e e))]
+      (is (= {:?w 10, :?t nil}
+             (-> e (ex-data) :bindings))))))
+
+(deftest test-local-scope-visible-in-test-filter
```

Comment:
I'm confused by this test case's name since it seems to be checking that the `:test` node is throwing an exception due to the `?t` value being nil right?

Reply:
I'll give it a more descriptive name and actually add additional assertions since there's a couple more things I want to check on this exception.